Resources/Prototypes/Body/Organs/human.yml

```
@@ -322,6 +322,8 @@
   name: kidneys
   description: "Filters toxins from the bloodstream."
   components:
+  - type: Organ # Shitmed
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)